### ISSUE TITLE
amp-facebook embed as video now include the card/caption

### ISSUE THREAD

**Issue by seomaz** [ampproject/amphtml#13748]
From today amp-facebook embed as video now include the caption/card information, for this you have the "post" options and "video" option is only for embed video without caption o card

**Comment by nainar** [comment]
Hi, this is an intentional change in behaviour to make sure that we were zooming in on videos correctly (See change [here](https://github.com/ampproject/amphtml/pull/13454/files)).

Given that I would suggest that you specify `data-embed-as` attribute to be `post`. This will make sure that we don't render the caption text.

**Comment by seomaz** [comment]
@nainar working fine on mobile, but in desktop using data-embed-ads = video and data-show-text = false its showing like a post not only video: example: https://www.cibercuba.com/videos/noticias/2018-03-01-u192519-e192519-s27061-maria-elvira-salazar-se-postula-oficialmente?1

**Comment by seomaz** [comment]
@nainar by the way, when you click on play the video always start without audio (mute options on)

**Comment by nainar** [comment]
Hi, sorry for the late reply. Can you send over an example page using `<amp-facebook-*>` components?

**Comment by seomaz** [comment]
You can see here: [https://www.cibercuba.com/videos/musica/2018-03-26-u194224-e196568-s27065-asi-fue-concierto-orishas-habana?nocache=1](https://www.cibercuba.com/videos/musica/2018-03-26-u194224-e196568-s27065-asi-fue-concierto-orishas-habana?nocache=1)

Using amp-facebook as video and the caption is always shown

**Comment by MtDalPizzol** [comment]
Same problem here. Any news on this? My code:

```
<amp-facebook
   width="480"
   height="320"
   layout="responsive"
   data-show-text="false" // I tried with and without it
   data-embed-as="video" // I also tried setting this to "post"
   data-href="{{ video.vid_facebook_url }}">
</amp-facebook>
```

**Comment by nainar** [comment]
Hi folks, 

I found the area that is causing the bug, I am in the process of uploading a patch to fix it. 

Regarding the video playing without sound @seomaz that is as determined by the Facebook API. 
Excerpt here regarding `data-autoplay`: 
```
The video will be played without sound (muted). People can turn on sound via the video player controls. This setting does not apply to mobile devices.
```
Link here: https://developers.facebook.com/docs/plugins/embedded-video-player

Thank you for your patience and using AMP. Sorry for the inconvenience caused.

**Comment by nainar** [comment]
Fixed in #14470
